examples/twilio-sms-connector/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <artifactId>twilio-demo-connector</artifactId>
```

Comment:
Can we name the artifactId to be consistent with other examples? So basically `einstein-bot-twilio-connector`.

https://github.com/forcedotcom/einstein-bot-channel-connector/blob/dabfbe210bace12680ab1a4249f54d493b2535a5/examples/twitter-connector/pom.xml#L12

Reply:
Agreed. Would it be ok to name it einstein-bot-twilio-sms-connector? It will be long but I added the "sms" keyword based on @relango 's recommendation to differentiate it from, say Twilio's voice/call or other future service connector.